zello-sdk-sample-ptt/src/main/java/com/zello/sdk/sample/ptt/PttActivity.java

```diff
@@ -11,233 +11,240 @@
 		Zello.getInstance().requestVitalPermissions(this);
 		_audio = Zello.getInstance().getAudio();
 
-        _connectChannelButton.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View v) {
-                onConnectChannelButtonPressed();
-            }
-        });
-
-        // PTT button push/release handler
-        _pttButton.setOnTouchListener(new View.OnTouchListener() {
-            @Override
-            public boolean onTouch(View v, MotionEvent event) {
-                int action = event.getAction();
-
-                if (action == MotionEvent.ACTION_DOWN) {
-                    Zello.getInstance().beginMessage();
-                } else if (action == MotionEvent.ACTION_UP || action == MotionEvent.ACTION_CANCEL) {
-                    Zello.getInstance().endMessage();
-                }
-
-                return false;
-            }
-        });
-
-        // Audio modes
-        _speakerButton.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View v) {
-                if (_audio != null) {
-                    _audio.setMode(com.zello.sdk.AudioMode.SPEAKER);
-                }
-            }
-        });
-
-        _earpieceButton.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View v) {
-                if (_audio != null) {
-                    _audio.setMode(com.zello.sdk.AudioMode.EARPIECE);
-                }
-            }
-        });
-
-        _bluetoothButton.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View v) {
-                if (_audio != null) {
-                    _audio.setMode(com.zello.sdk.AudioMode.BLUETOOTH);
-                }
-            }
-        });
-
-        updateAudioMode();
-    }
-
-    @Override
-    protected void onDestroy() {
-        super.onDestroy();
-
-        Zello.getInstance().unsubscribeFromEvents(this);
-        Zello.getInstance().unconfigure();
-    }
-
-    @Override
-    protected void onResume() {
-        super.onResume();
+		_connectChannelButton.setOnClickListener(new View.OnClickListener() {
+			@Override
+			public void onClick(View v) {
+				onConnectChannelButtonPressed();
+			}
+		});
+
+		// PTT button push/release handler
+		_pttButton.setOnTouchListener(new View.OnTouchListener() {
+			@Override
+			public boolean onTouch(View v, MotionEvent event) {
+				int action = event.getAction();
+
+				if (action == MotionEvent.ACTION_DOWN) {
+					Zello.getInstance().beginMessage();
+				} else if (action == MotionEvent.ACTION_UP || action == MotionEvent.ACTION_CANCEL) {
+					Zello.getInstance().endMessage();
+				}
+
+				return false;
+			}
+		});
+
+		// Audio modes
+		_speakerButton.setOnClickListener(new View.OnClickListener() {
+			@Override
+			public void onClick(View v) {
+				if (_audio != null) {
```

Comment:
Minor, but if you could early return all of these OnClickListeners that would be good

Reply:
Call me old fashioned, but I think this takes the rule too far.